wdl2cwl/main.py

```diff
@@ -73,23 +73,20 @@ def load_wdl_workflow(self, obj: WDL.Tree.Workflow) -> cwl.Workflow:
         outputs: List[cwl.WorkflowOutputParameter] = []
         wf_steps: List[cwl.WorkflowStep] = []
         wf_name = obj.name
-        wf_description = obj.meta["description"]
+        wf_description = obj.meta["description"] if "description" in obj.meta else None
         for call in obj.body:
-            call_name = call.name  # type: ignore
             callee = call.callee  # type: ignore
-            namespace, _ = call.callee_id  # type: ignore
+            callee_id = ".".join(call.callee_id)  # type: ignore
```

Comment:
callee_id is a list of the namespace of where the call is from and the call itself. This works even when there is no task imported